app/models/user/academics/registration.rb

```diff
@@ -27,6 +31,17 @@ def law?
         career_code == LAW
       end
 
+      def academic_levels
+        data['academicLevels'].map do |level|
+          ::User::Academics::Level.new(level)
+        end
+      end
+
+      def career_preferred_level
+        career_preferred_type_code = career_code == 'LAW' ? 'EOT' : 'BOT'
+        academic_levels.find {|al| al.type_code == career_preferred_type_code }
+      end
+
```

Comment:
If we create a `::User::Academics::Levels` object we can contain the knowledge about the meanings of 'EOT' and 'BOT' in a place that makes sense as well as making the API able to be used like `user.latest_registration_term.preferred_level.description`.

Not a deal-breaker, just an idea for later

Reply:
@developish - I'm creating levels, but because it relies on the career code of the registration, it still seems more applicable to keep this functionality within `User::Academics::Registration`, as it's the preferred level based on it's career code.

I am shortening the name to `preferred_level` though.